hawk/app/views/resources/show.html.haml

```diff
@@ -10,121 +10,102 @@
-                      %td= content_tag :code, op["name"]
-                      %td= content_tag :code, op["timeout"] || "-"
-                      %td= content_tag :code, op["interval"] || "-"
+                  %td= content_tag :code, name
+                  %td= content_tag :code, value
+      .panel.panel-default
+        .panel-heading#r-metacontrol{role: :tab}
+          %h4.panel-title
+            %a{role: :button, data: {toggle: :collapse}, href: "#r-metas", aria: {expanded: true, controls: "r-metas"}}
+              = _("Meta Attributes")
+              .pull-right
+                %span.caret
+        #r-metas.panel-collapse.collapse.in{role: :tabpanel, aria: {labelledby: "r-metacontrol"}}
+          %table.table
+            %tr
+              %th= _("Name")
+              %th= _("Value")
+            - @resource.meta.each do |name, value|
+              %tr
+                %td= content_tag :code, name
+                %td= content_tag :code, value
+
+      - if @resource.ops.any?
+        .panel.panel-default
+          .panel-heading#r-opcontrol{role: :tab}
+            %h4.panel-title
+              %a{role: :button, data: {toggle: :collapse}, href: "#r-ops", aria: {expanded: false, controls: "r-ops"}}
+                = _("Operations")
+                .pull-right
+                  %span.caret
+          #r-ops.panel-collapse.collapse{role: :tabpanel, aria: {labelledby: "r-opcontrol"}}
+            %table.table
+              %tr
+                %th= _("Name")
+                %th= _("Timeout")
+                %th= _("Interval")
+              - @resource.ops.each do |id, op|
+                %tr
+                  %td= content_tag :code, op["name"]
+                  %td= content_tag :code, op["timeout"] || "-"
+                  %td= content_tag :code, op["interval"] || "-"
 
-        - if @resource.running_on.any?
-          .panel.panel-default
-            .panel-heading#r-instances{role: :tab}
-              %h4.panel-title
-                %a{role: :button, data: {toggle: :collapse, parent: "#r-info"}, href: "#r-instances-body", aria: {expanded: false, controls: "r-instances-body"}}
-                  = _("Instances")
-                  .pull-right
-                    %span.caret
-            #r-instances-body.panel-collapse.collapse{role: :tabpanel, aria: {labelledby: "r-instances"}}
-              .panel-body
-                %table.table
-                  %tr
-                    %th= _("Node")
-                    %th= _("State")
-                  - @resource.running_on.each do |node, state|
-                    %tr
-                      %td= content_tag :code, node
-                      %td= content_tag :code, state || "-"
+      - if @resource.running_on.any?
+        .panel.panel-default
+          .panel-heading#r-instances{role: :tab}
+            %h4.panel-title
+              %a{role: :button, data: {toggle: :collapse}, href: "#r-instances-body", aria: {expanded: false, controls: "r-instances-body"}}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [140/80]